pmdarima/arima/tests/test_arima.py

```diff
@@ -16,14 +16,34 @@
 from sklearn.externals import joblib
 
 from statsmodels import api as sm
-from statsmodels.tsa.base.tsa_model import TimeSeriesModelResults
 import pandas as pd
 
 import warnings
 import pickle
 import pytest
 import time
 import os
+import shutil
+
+
+# test images directories
+travis = os.environ.get("TESTING_ON_TRAVIS", "false").lower() == "true"
+
+if not travis:
+
+    # Only do this setup if we are not running on Travis CI.
+    from matplotlib.pyplot import savefig
+    from matplotlib.testing.exceptions import ImageComparisonFailure
+    import matplotlib._png as _png
+
+    test_images_input_path = \
+        os.path.join('pmdarima', 'arima', 'tests', 'input_images')
+    test_images_output_path = \
+        test_images_input_path.replace('input_images', 'output_images')
+
+    # make output directory if it doesn't exist
+    if not os.path.exists(test_images_output_path):
+        os.makedirs(test_images_output_path)
```

Comment:
Can you put this whole `if not travis` block inside the other? And this portion in particular should go inside the `try/finally`